jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I made this dir...
In my understanding, config dir has task env config.
the information is important for the task, and not often changed.